.travis.yml

```diff
@@ -92,7 +92,7 @@ install:
   # Install ldoc for building docs.
   - |
     if [ "$BUILD_APIDOC" == "true" ]; then
-      travis_retry sudo luarocks install ldoc 1.4.4
+      travis_retry sudo luarocks install ldoc 1.4.6
```

Comment:
How does this help with speed and accuracy? Why is ldoc still pinned to a specific version?